source/galaxy-database-mongodb-general.md

```diff
@@ -0,0 +1,90 @@
+---
+title: MongoDB - General
+order: 14
+description: Information about supported regions, MongoDB versions, types, hardware resources, storage management, disk type, backup policies, logs and metrics, and users for Galaxy Databases.
+---
+
+<h2 id="supported-regions">Supported regions</h2>
+
+The Galaxy Databases offer is available in the following regions:
+
+- VIR (Virginia, USA)
+- FRA (Frankfurt, Germany)
+- SYD (Sydney, Australia)
+
+<h2 id="mongodb-versions">MongoDB versions</h2>
+
+The Galaxy Databases offer supports the following MongoDB versions:
+
+- MongoDB 4.0.19
+- MongoDB 4.2.23
+- MongoDB 4.4.17
+- MongoDB 5.0.14
+- MongoDB 6.0.5
+- MongoDB 7.0.4
+
+<h2 id="types">Types</h2>
+
+- **Standalone**: A single instance of MongoDB that operates independently.
+- **ReplicaSet**: A set of MongoDB instances that offers high availability and data replication.
```

Comment:
maybe also add that the Standalone does not support Oplog and only Replicaset does